webapp/components/test/test-file-results-table-terse.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
```

Comment:
Note: This is a migration of the tests that cover the sub-component of `test-file-results`, `test-file-results-table-terse`.